client/components/GAuth.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { GoogleLogin } from 'react-google-login'
+import GoogleButton from 'react-google-button'
+import { refreshTokenSetup } from './utils/refreshToken'
+import getConfig from "next/config";
+
+
+const { publicRuntimeConfig } = getConfig();
+
+function Login({ parentCallback }) {
+
+    const onSuccess = (res) => {
+        //Sees profile object that google returns
+        //console.log('[Login Success] currentUser: ', res.profileObj);
```

Comment:
I guess these comments with unused code or `console.log` should be removed (there are more throughout the code).